invenio_accounts/models.py

```diff
@@ -87,3 +89,17 @@ def validate_ip(self, key, value):
         if value == 'untrackable':  # pragma: no cover
             value = None
         return value
+
+
+class SessionActivity(db.Model, Timestamp):
+    """User Session Activity model."""
+
+    __tablename__ = "accounts_user_session_activity"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    user_id = db.Column(db.Integer, db.ForeignKey(User.id))
+
+    sid_s = db.Column(db.String(255), unique=True, index=True)
```

Comment:
Possibly the whole thing means `session id serialized` (the last `s` standing for `serialized`), as the `sid_s` value is unserialized to produce the actual session.
It is, however, _not_ the session's id and naming the field `session_id` would be confusing (looking through the code you'd then see that the value stored here as `session_id` is referred to as `sid_s` in `flask_kvsession`, and said value gets unserialized to produce the session object which contains the session id).

The field contains the session's key value as stored in `flask_kvsession`'s sessionstore.
I've named this field `sid_s` so that it corresponds with the naming used in `flask_kvsession` to avoid confusion.